bootstrap/bootstrap.go

```diff
@@ -1370,10 +1370,12 @@ func (b *Bootstrap) defaultCommandPhase() error {
 
 		cmdToExec = batchScript
 	} else if commandIsScript {
-		// Make script executable
-		if err = addExecutePermissionToFile(pathToCommand); err != nil {
-			b.shell.Warningf("Error marking script %q as executable: %v", pathToCommand, err)
-			return err
+		if b.Config.CommandEval {
```

Comment:
Could I be difficult and ask that you add a comment with some context around why we only do this in CommandEval and why it's a security risk? Will help us remember in 12 months time when we look in on it.